### PR TITLE
Add test for "mov" and "(mov|add|adds|sub|subs)_imm"

### PR DESCRIPTION
**Pull request by bkmgit**
Additional tests for arithmetic and mov to be used in CI.
Addressing https://github.com/fujitsu/xbyak_aarch64/issues/27
Unclear if the codes should be refactored.

### CODE REVIEW

**Comment by kawakami-k** [comment]
Thank you for the PR, but the tesfs of `*_imm` have already been implemented as [mov_imm.cpp](https://github.com/fujitsu/xbyak_aarch64/blob/9832b380e7a85a41635a9de84d50bbf54f44e9b6/test/mov_imm.cpp) and [add_sub_adds_subs_imm.cpp](https://github.com/fujitsu/xbyak_aarch64/blob/9832b380e7a85a41635a9de84d50bbf54f44e9b6/test/add_sub_adds_subs_imm.cpp).

These files should be compiled and executed in `test_all.sh`.

**Comment by bkmgit** [comment]
`test_all.sh` calls `test_integrated.sh` which compiles and executes  [mov.cpp](https://github.com/fujitsu/xbyak_aarch64/blob/9832b380e7a85a41635a9de84d50bbf54f44e9b6/test/mov_imm.cpp), [mov_imm.cpp](https://github.com/fujitsu/xbyak_aarch64/blob/9832b380e7a85a41635a9de84d50bbf54f44e9b6/test/mov_imm.cpp) and [add_sub_adds_subs_imm.cpp](https://github.com/fujitsu/xbyak_aarch64/blob/9832b380e7a85a41635a9de84d50bbf54f44e9b6/test/add_sub_adds_subs_imm.cpp). Should the contents of `test_integrated.sh` be put in `test_nm.sh` which is what calls the other tests? A few changes are needed to run these tests compared to the other tests in `test_nm.sh`.

File: test/test_integrated.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+${CXX} ${TEST_FILE}.o -o ${TEST_FILE} -L ../lib -lxbyak_aarch64
+
+if [ $? != 0 ] ; then
+    dumpNG "Compiling binary."
+    exit 1
+fi
+
+
+# Output 
+./${TEST_FILE} > a.asm
```
Comment (kawakami-k) [inline test/test_integrated.sh:105]: Could you please change this line?
```
${EMULATOR} ./${TEST_FILE} > ${TEST_FILE}.log 2>&1
```

File: test/test_integrated.sh [hunk 2]
```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+#*******************************************************************************
+# Copyright 2019-2021 FUJITSU LIMITED
```
Comment (kawakami-k) [inline test/test_integrated.sh:3]: Could you please change this line?
```
# Copyright 2019-2022 FUJITSU LIMITED
```

**Review by kawakami-k — CHANGES_REQUESTED** [review]
Thank you for the PR!
 Could you please chane three miner points?
- Two lines for `Copyright 201-2022 FUJITSU LIMITED`.
- `${EMULATOR} ./${TEST_FILE} > ${TEST_FILE}.log 2>&1`

File: test/test_integrated.sh (hunk 2 above)
Comment (bkmgit) [reply test/test_integrated.sh]: Done.

File: test/test_integrated.sh (hunk 1 above)
Comment (bkmgit) [reply test/test_integrated.sh]: Done.

**Comment by kawakami-k** [comment]
> `test_all.sh` calls `test_integrated.sh` which compiles and executes [mov.cpp](https://github.com/fujitsu/xbyak_aarch64/blob/9832b380e7a85a41635a9de84d50bbf54f44e9b6/test/mov_imm.cpp), [mov_imm.cpp](https://github.com/fujitsu/xbyak_aarch64/blob/9832b380e7a85a41635a9de84d50bbf54f44e9b6/test/mov_imm.cpp) and [add_sub_adds_subs_imm.cpp](https://github.com/fujitsu/xbyak_aarch64/blob/9832b380e7a85a41635a9de84d50bbf54f44e9b6/test/add_sub_adds_subs_imm.cpp). Should the contents of `test_integrated.sh` be put in `test_nm.sh` which is what calls the other tests? A few changes are needed to run these tests compared to the other tests in `test_nm.sh`.

Sorry, I made a mistake. You're right.
Thank you for the contribution.

**Comment by bkmgit** [comment]
Welcome. Thanks for the code.

**Pull request merged**